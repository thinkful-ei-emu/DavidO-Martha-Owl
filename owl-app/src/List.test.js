import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import PeopleList from './List.js';


describe('<List />', () =>{
  it('renders without crashing', ()=> {
    const div = document.createElement('div');
    ReactDOM.render(<PeopleList />, div);
    ReactDOM.unmountComponentAtNode(div);

  });
});

it('renders the UI as expected', () => {
  const snapshot = renderer
    .create(<PeopleList />)
    .toJSON();
  expect(snapshot).toMatchSnapshot();  
});