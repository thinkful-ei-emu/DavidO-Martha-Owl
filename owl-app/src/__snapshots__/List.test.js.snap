// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="ListofPeople"
>
  <ul>
    <div>
      <img
        alt="peoples faces"
        src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
      />
      <li>
         
        Koren Templeton
          
      </li>
      <p>
         
        On Stage
          
      </p>
    </div>
    <div>
      <img
        alt="peoples faces"
        src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
      />
      <li>
         
        Caty Flucker
          
      </li>
      <p>
         
          
      </p>
    </div>
    <div>
      <img
        alt="peoples faces"
        src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
      />
      <li>
         
        Axe Kubicka
          
      </li>
      <p>
         
          
      </p>
    </div>
    <div>
      <img
        alt="peoples faces"
        src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
      />
      <li>
         
        Frank Runciman
          
      </li>
      <p>
         
          
      </p>
    </div>
    <div>
      <img
        alt="peoples faces"
        src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
      />
      <li>
         
        Ashla Attwool
          
      </li>
      <p>
         
        On Stage
          
      </p>
    </div>
  </ul>
</div>
`;
